new-lamassu-admin/src/pazuz/pages/ATMWallet/ATMWallet.js

```diff
@@ -0,0 +1,196 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Any reason for this?

Reply:
Just clutter from development, nice catch. I'll remove it!